examples/mobile/HomeApp/webclip/restaurant/hotel-breakfast-wc.js

```diff
@@ -1,11 +1,184 @@
-/* global tau */
+const tmpl = document.createElement("template");
+
+tmpl.innerHTML = `
```

Comment:
I just have a question.
There are the css and html code in the js file.
A general developer who distributes web clips, he have to include the css/html code into js fie??
Or it's just temporary?

Reply:
how about separating them in future like in
https://stackoverflow.com/questions/55080103/how-to-separate-web-components-to-individual-files-and-load-them

Reply:
This commit is early draft. I try verify different ways to achieving the goal.

Reply:
Developer who distribute web clip can do as he needs. In my case I wanted to add template to js file. This plugin has been realized by Custom Element but other Web Clips as "TV Remote Control" has been made as html and css.

At the moment plugin need `html` file eg. `index.html` with
`<div class="ui-card"></div>` element.